### DISCUSSION TITLE
收集错误上报时，错误发生的文件指向了umi.xxxx.js

### DISCUSSION THREAD

**Discussion by LanJamRom** [umijs/umi#9790] [Q&A] [answered]
按正常来说不应该指向按页拆包出来的js文件吗？
<img width="497" alt="image" src="https://user-images.githubusercontent.com/41603622/202140728-4511aec1-046c-4e3a-9df9-da6d1911bb4b.png">

**Comment by fz6m** [comment, marked as answer]
错误栈是往上抛的，第一层确实是分包出来的 js 文件

**Reply by LanJamRom** [reply]
有解法吗，这样搜集到的lineno、colno经sourcemap反解的是umi.xxx.js 反解结果指到react去了。。。

**Reply by fz6m** [reply]
你们收集报错的 sdk 不会把全部的 stack 都上报吗，有全部 stack 的话，就可以看出来在哪个地方报错了，在监控平台上看到的应该和 console 一样。
